packages/web-components/src/design-tokens.ts

```diff
@@ -0,0 +1,571 @@
+        neutralPalette.getValueFor(element),
+        baseLayerLuminance.getValueFor(element),
+        neutralFillCardDelta.getValueFor(element),
+      ),
+    ),
+);
+
+export const neutralLayerFloating = create<SwatchRGB>('neutral-layer-floating').withDefault((element: HTMLElement) =>
+  DI.getOrCreateDOMContainer(element).get(NeutralLayerFloating)(element),
+);
+
+// Neutral Layer L1
+export const NeutralLayerL1 = DI.createInterface<(element: HTMLElement) => SwatchRGB>('neutral-layer-L1', builder =>
+  builder.instance((element: HTMLElement) =>
+    neutralLayerL1Algorithm(neutralPalette.getValueFor(element), baseLayerLuminance.getValueFor(element)),
+  ),
+);
+
+export const neutralLayerL1 = create<SwatchRGB>('neutral-layer-L1').withDefault((element: HTMLElement) =>
+  DI.getOrCreateDOMContainer(element).get(NeutralLayerL1)(element),
+);
+
+// Neutral Layer L2
+export const NeutralLayerL2 = DI.createInterface<(element: HTMLElement) => SwatchRGB>('neutral-layer-L2', builder =>
+  builder.instance((element: HTMLElement) =>
+    neutralLayerL2Algorithm(
+      neutralPalette.getValueFor(element),
+      baseLayerLuminance.getValueFor(element),
+      neutralFillCardDelta.getValueFor(element),
+      neutralFillRestDelta.getValueFor(element),
+      neutralFillHoverDelta.getValueFor(element),
+      neutralFillActiveDelta.getValueFor(element),
+    ),
+  ),
+);
+
+export const neutralLayerL2 = create<SwatchRGB>('neutral-layer-L2').withDefault((element: HTMLElement) =>
+  DI.getOrCreateDOMContainer(element).get(NeutralLayerL2)(element),
+);
+
+// Neutral Layer L3
+export const NeutralLayerL3 = DI.createInterface<(element: HTMLElement) => SwatchRGB>('neutral-layer-L3', builder =>
+  builder.instance((element: HTMLElement) =>
+    neutralLayerL3Algorithm(
+      neutralPalette.getValueFor(element),
+      baseLayerLuminance.getValueFor(element),
+      neutralFillCardDelta.getValueFor(element),
+      neutralFillRestDelta.getValueFor(element),
+      neutralFillHoverDelta.getValueFor(element),
+      neutralFillActiveDelta.getValueFor(element),
+    ),
+  ),
+);
+
+export const neutralLayerL3 = create<SwatchRGB>('neutral-layer-L3').withDefault((element: HTMLElement) =>
+  DI.getOrCreateDOMContainer(element).get(NeutralLayerL3)(element),
+);
+
+// Neutral Layer L4
+export const NeutralLayerL4 = DI.createInterface<(element: HTMLElement) => SwatchRGB>('neutral-layer-L4', builder =>
+  builder.instance((element: HTMLElement) =>
+    neutralLayerL4Algorithm(
+      neutralPalette.getValueFor(element),
+      baseLayerLuminance.getValueFor(element),
+      neutralFillCardDelta.getValueFor(element),
+      neutralFillRestDelta.getValueFor(element),
+      neutralFillHoverDelta.getValueFor(element),
+      neutralFillActiveDelta.getValueFor(element),
+    ),
+  ),
+);
+
+export const neutralLayerL4 = create<SwatchRGB>('neutral-layer-L4').withDefault((element: HTMLElement) =>
+  DI.getOrCreateDOMContainer(element).get(NeutralLayerL4)(element),
```

Comment:
I noticed that we have a lot of use of `DI.getOrCreateDOMContainer()` throughout this file, which looks to see whether the current element has a container defined on it and if not, creates one for that element. Is this what we want or did we mean to use `DI.findResponsibleContainer()` instead. Just want to make sure. /cc @nicholasrice

Reply:
@nicholasrice to advise on this. This is the current suggested implementation based on FAST's vNext component strategy.

Reply:
@EisenbergEffect Either would work I suppose. If `findResponsibleContainer()` is more performant than we can certainly use that; this is just trying to resolve the recipe so that authors can override it if necessary.

Reply:
Well, it's a matter of what behavior you want. `DI.getOrCreateDOMContainer()` checks to see if there's a container defined directly on the passed in element. If not, it creates one (a child container) on that element. So, that effects all downstream resolutions in the DOM. On the other hand `DI.findResponsibleContainer()` dispatches the dom event to find a parent container in the DOM. If no container is found, it creates a container on the body and returns that.

Reply:
I think you probably want `DI.findResponsibleContainer()`. You don't want to create a new container on the element, but simply want to find the requested service in the container hierarchy. Is that correct?

Reply:
I just made an update here to leverage this (also leveraging in tooltip), I think this is what we'd want in card as well. Thanks @EisenbergEffect!